docs/quick-start/quick-start-online-debug.md

```diff
@@ -1,25 +1,21 @@
-<h1 id="debug-ide-qs">Debugging the quick start</h1>
+<h1 id="debug-ide-qs">Debugging with the Online Compiler</h1>
```

Comment:
There is a lot of similarities with cli debug. I think it would be better to combine the two pages.
What do you think @iriark01 ?

Reply:
I personally agree - the quick start pages are rather brief, maybe CLI and online compiler can go into different sections of one page

Reply:
People using one tool don't like having to read through instructions/info for another tool. And sometimes they don't even realise it's there so they assume it's missing.

Reply:
@iriark01 I understand if the instructions are quite different and long. In this case it is rather short and it would remove the duplication. I thought you combined instructions for blinky bare metal ; this would be another example of combined instructions.

Reply:
The current documentation layout on the website
```
Quick start
|- Introduction
|- Online with the online compiler
    |- Importing and compiling the code
    |- Debugging with the online compiler *
|- Offline with Mbed CLI
    |- Setting up and compiling the code
    |- Debugging with Mbed CLI *
|- Further reading

* what this PR touches
```
Maybe, we can rename both "Debugging with ..." to "Exporting to a desktop IDE", and factor out "Using printf" (and how to drag & drop an image) into a separate page "Debugging"? It would still look natural in my personal opinion.

Reply:
I combined bare metal and the full profile, but not into the same tool. The tools are seen as very separate beasts - Mbed Studio actually has its own site.

I would argue that the problem here is that there's no real value to having either of these pages in the quick start. Debugging and exporting are both covered in much greater detail elsewhere in the docs.

@LDong-Arm please can you use a branch on this repo rather than a fork? That will generate a preview of the docs site with your changes, which will make this discussion much easier.